code/obj/machinery/computer/card.dm

```diff
@@ -70,8 +73,9 @@
 
 		if(src.loc == user)
 			user.drop_from_slot(src)
-		user.visible_message("<span class='alert'>[user] unfolds the foldable portable idendification computer from a briefcase!</span>")
+		user.visible_message("<span class='alert'>[user] unfolds the foldable portable idendification computer from a briefcase, there was also a pinpointer inside, how handy!</span>")
```

Comment:
This is a separate item, that's a copy of the briefcase the computer can turn into. It serves the dual purpose of giving a item to use to deploy the computer from for the first time that isn't tied to the computer, so it can be spawned into the map in a undeployed state, and makes it so there's only one pinpointed made! :)